vpr/src/route/rr_graph_util.cpp

```diff
@@ -75,3 +80,76 @@ int seg_index_of_sblock(int from_node, int to_node) {
         return OPEN; //Should not reach here once thrown
     }
 }
+
+void reorder_rr_graph_nodes(const t_router_opts& router_opts) {
```

Comment:
Comment what the routine does, and any invariants expected (e.g. I assume that the rr-graph should be built before calling this, but there should not be a routing created or loaded as it would depend on the rr_node indices in the current graph).  The rr-graph nodes are renumbered in place. Any other conditions where this shouldn't be called?

Reply:
Done.